### PR TITLE
Add designate plugin

### PR DESCRIPTION
**Pull request by gboutry**
Depends-On:
- [x] https://github.com/canonical/sunbeam-terraform/pull/7

### CODE REVIEW

File: sunbeam-python/sunbeam/plugins/dns/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# dns service
```
Comment (pmatulis) [inline sunbeam-python/sunbeam/plugins/dns/README.md:1]: DNS

File: sunbeam-python/sunbeam/plugins/dns/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+This plugin provides a DNS service for Sunbeam. It based on [Designate](https://docs.openstack.org/designate/latest/), a DNS service for OpenStack and Bind.
```
Comment (pmatulis) [inline sunbeam-python/sunbeam/plugins/dns/README.md:3]: I think there are multiple DNS backends possible, and we have chosen Bind?

Comment (gboutry) [reply sunbeam-python/sunbeam/plugins/dns/README.md:3]: Yes, current deployment is only Bind

File: sunbeam-python/sunbeam/plugins/dns/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+sunbeam dns address
+```
+
+## Removal
+
+To remove the plugin, run:
+
+```bash
+sunbeam disable dns
+```
```
Comment (gnuoy) [inline sunbeam-python/sunbeam/plugins/dns/README.md:46]: I think we should give a developer pointers here on how to get started if they wanted to make a change.  They would need to know what charms are going to be deployed and perhaps highlight what those charm are responsible for. We don't want to duplicate the charms documentation so it only needs to be high level, we can then also include pointers to the charms and rocks code repos.

**Review by gnuoy — CHANGES_REQUESTED** [review]
In terms of the README

**Review by gnuoy — APPROVED** [review]

**Pull request merged**